Client/config/info_loader.py

```
@@ -0,0 +1,54 @@
+    else:
+        try:
+            with open(path, "r") as f:
+                content = f.read()
+        except Exception as e:
+            logger.exception("Error reading info.txt: %s", e)
+            return ""
+
+    # Inject secrets from environment variables
+    # We replace any existing Machine Password line or append if not present
+    # Note: We redact the actual password when returning info to be safe
+    password_line = "Machine Password: [REDACTED]"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "password_line"

(S105)